README.md

```diff
@@ -42,10 +42,19 @@ Download Nightly releases
 
 ## Browser Support
 
-- [Chrome](https://chrome.google.com/webstore/detail/alby/iokeahhehimjnekafflcihljlcjccdbe) (Opera, Brave, and all Chromium based browsers)
-- [Firefox](https://addons.mozilla.org/en-US/firefox/addon/alby/)
+Alby supports
+
+- All [Chromium based browsers](https://en.wikipedia.org/wiki/Chromium_(web_browser)#Browsers_based_on_Chromium) - Chrome Opera, Brave etc.
+- Firefox
 - more coming soon...
 
+## Installation
+
```

Comment:
now that we have this section. 
can we move the `## Try out the most recent version` to here. 

So under `## Installation` people find the links to the latest official releases in the stores and to the nightly builds.

Reply:
can we say official releases as stable versions? and nightly releases as a most recent alpha version

Reply:
yep

Reply:
done